src/components/organisms/Sections/FAQ/index.jsx

```diff
@@ -0,0 +1,61 @@
+//> React
+// Contains all the functionality necessary to define React components
+import React from 'react';
+
+//> MDB
+// "Material Design for Bootstrap" is a great UI design framework
+import {
+    MDBContainer,
+    MDBRow,
+    MDBCol,
+    MDBIcon,
+} from 'mdbreact';
+
+const items = [
+    {
+        icon: "leaf",
+        heading: "Wie brät man Lorem Ipsum?",
+        text: "Lorem Ipsum dolor sit amet",
+        link: ""
+    },
+    {
+        icon: "leaf",
+        heading: "Schmeckt dieser Kaffee gleich?",
```

Comment:
This doesn't sound like a proper German question. xD

Reply:
Content is not the objective of this pull request.